public/chassis.css

```diff
@@ -0,0 +1,392 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700;800&display=swap");
+
+* {
+  font-family: "Open Sans", sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --color-1: #1f1f27;
+  --color-2: #f3f4f6;
+}
+
+section {
+  padding-bottom: 1rem;
+  border-bottom: 5px solid red;
+}
+
+.avatar-sm {
+  width: 30px;
+  height: 30px;
+  border-radius: 50%;
+}
+
+.avatar-md {
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+}
+
+.avatar-big {
+  width: 80px;
+  height: 80px;
+  border-radius: 50%;
+}
+
+.alert {
+  width: fit-content;
+  border: 1px solid #9ca3af;
+  padding: 0.2rem 0.5rem;
+  background-color: #f9fafb;
+  font-size: 0.9em;
+}
+
+.alert-success {
+  border: 1px solid #34d399;
+  background-color: #eff6ff;
+}
+
+.alert-warn {
+  border: 1px solid #f87171;
+  background-color: #fef2f2;
+}
+
+.icon {
+  width: 40px;
+  height: 40px;
+}
+
+.icon > svg {
+  width: 100%;
+  height: auto;
+}
+
+.badge-icon {
+  font-size: 0.6em;
+  position: relative;
+  bottom: 50px;
+  left: 28px;
+  font-weight: bold;
+  background-color: red;
+  width: 14px;
+  height: 14px;
+  color: white;
+  text-align: center;
+  font-weight: bold;
+  border-radius: 50%;
+}
+
+.badge-avatar {

```

Comment:
You can create a separate badge class so that you don't have to repeat the styles in different badges

Reply:
Great suggestions! Thanks.
I'll fix them.

Reply:
@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700;800&display=swap")

It is not a good practice to use import inside CSS files because it will wait for the import to complete and code below the import won't get load, so it may create issues

Correct me If I am wrong.

Reply:
Changes:
1. I have changed the badge for the avatar to red, green and off-white.
2. I have made some utility classes to reduce the repetition.